draft-ietf-lsr-isis-fast-flooding.xml

```diff
@@ -784,7 +757,7 @@
           current LSPTxRate.</t>
 
           <t>The flow control algorithm MUST NOT assume the receive

```

Comment:
Please replace all occurrences in this section  of "flow control" with "congestion control".

Reply:
OK. I took the liberty to also apply the chane to the time. Please correct me if I'm wrong.